frontend/src/components/Auth.js

```diff
@@ -0,0 +1,133 @@
+    this.login = this.login.bind(this);
+    this.signUp = this.signUp.bind(this);
+    this.setCredentials = this.setCredentials.bind(this);
+  };
+
+  changeState() {
+    const currentValue = this.state.login;
+    this.setState({login: !currentValue});
+  };
+
+  logInBackGround() {
+    const loginFlag = this.state.login;
+    const background = loginFlag? '#FFFFFF' : '#A463F2';
+    return background;
+  };
+
+  signUpBackGround () {
+    const loginFlag = this.state.login;
+    const background = loginFlag? '#A463F2' : '#FFFFFF';
+    return background;
+  };
+
+  button() {
+    const loginFlag = this.state.login;
+    var valueToReturn = loginFlag? 
+      (<a className="f6 link dim ph3 pv2 mb2 dib white bg-purple" href="#0" onClick={this.login}>
+        Login
+      </a>) : 
+      (<a className="f6 link dim ph3 pv2 mb2 dib white bg-purple" href="#0" onClick={this.signUp}>
+        Sign Up
+      </a>);
+    return (<div className="ph3 flex justify-center h-40">{valueToReturn}</div>);
+  };
+
+  login() {
+    login(this.state.email, this.state.password, (user) => {
+      const requestHeader = {
+        email: user.email, 
+        token: user.refreshToken
+      };
+      
+      this.props.loginUser(requestHeader);
+      this.clearState();
+    });
+  };
+
+  clearState() {
+    this.setState({
+      email: '',
+      password: '',
+      login: true
+    });
+  };
+
+  signUp() {
+    create(this.state.email, this.state.password, user => {
+      const body = {email: user.email};
+      this.props.createUser(body);
+      this.clearState();
+    });
+  };
+
+  setCredentials = field => event => {
+    this.setState({
+      [field]: event.target.value
+    });
+  };
+
+  render() {
+    const loginFlag = this.state.login;
+    const loginBg = this.logInBackGround();
+    const singUpBg = this.signUpBackGround();
+    
+    return (
+      <div style={{'height': '200px', 'width': '300px'}}>
+        <article className="ba b--purple content-center h-100">
+          <div className="flex justify-center h-20" style={{'backgroundColor': "#FFFFFF"}}>
+            <div className="h-100 w-50 tc">
+              <div style={{'height': '40px', 'backgroundColor': loginBg}}
+                onClick={this.changeState}>
+                <span style={{'fontSize': '25px', 'color': loginFlag? '#A463F2' : '#FFFFFF'}}>
+                  Log In
+                </span>
+              </div>
+            </div>
+            <div className="h-100 w-50 tc">
+              <div style={{'height': '40px', 'backgroundColor': singUpBg}}
+                onClick={this.changeState}>
+                <span style={{'fontSize': '25px', 'color': loginFlag? '#FFFFFF' : '#A463F2'}}>Sign Up</span>
+              </div>
+            </div>
+          </div>
+          <div className="flex justify-center h-40">
+            <form className="flex flex-column pa4 black-80">
+              <div className="measure" style={{'borderRadius': '50px'}}>
```

Comment:
There is a tachyons class that deals with rounded borders. It is br-X, where X is how much rounded should it be, see reference for more details.